package.json

```diff
@@ -77,6 +77,7 @@
     "less": "3.12.2",
     "less-loader": "^7.0.1",
     "less-vars-to-js": "^1.3.0",
+    "lessc": "^1.0.2",
```

Comment:
Trenger vi lessc på rot når hver pakke har ansvar for å bygge seg selv?

Reply:
Siden dependencies i pakker ikke blir lastet ned med yarn install må den være på root, men tenker vi kan løse dette med  `npx lessc` ? @andnorda